app/presenters/document_presenter.rb

```diff
@@ -0,0 +1,6 @@
+# frozen_string_literal: true
+
+class DocumentPresenter < Jsonite
+  property :id
+  property :url
```

Comment:
Thanks Will! This is super helpful. 

There is one other property that we want, perhaps we can call it `description`. We'll be using it to list the language of the document. @jjfreund -- I believe we decided it would be a string right, rather than an enum?

Reply:
If we use an enum, would it be better to name the field "language" or something like that? Description sounds like it applies to much more than just the language of the doc

Reply:
That is a good point. I would agree with that. I think we ultimately decided to go with a string and to call the field `description` so that the model could potentially be used to represent other sorts of data.

Reply:
Document already has the following properties. They should be added to the presenter.

      t.string :name
      t.string :description

Reply:
Updated, also added back the migration file that was accidentally deleted.